homeassistant/components/xiaomi_miio/services.yaml

```diff
@@ -373,3 +373,29 @@ vacuum_clean_segment:
       example: "[1,2]"
       selector:
         object:
+
+vacuum_reset_consumable:
+  name: Vacuum reset consumable
+  description: Reset status of the specified consumable.
+  target:
+    entity:
+      integration: xiaomi_miio
+      domain: vacuum
+  fields:
+    consumable:
+      name: Consumable
+      description: Consumable.
+      example: "Main brush"
+      required: true
+      selector:
+        select:
+          options:
+            - label: "Main brush"
+              value: "main_brush_work_time"
+            - label: "Side brush"
+              value: "side_brush_work_time"
+            - label: "Filter"
+              value: "filter_work_time"
+            - label: "Sensor dirty"
+              value: "sensor_dirty_time"
+            # TODO: implement `cleaning_brush_work_times` and `strainer_work_times` (any others?) in miio
```

Comment:
@zry98 please remove the comment with the TODO 👍

linter fails specifically because the indentation, but that kind of comments are not common practice around here 😜

Reply:
Removed 👌